Bottom: 2cfd1213602b08cffdeaa70b35ffe759618c3cec
Top:    9e3e037135928d4420496bf19acfd9b674f415cc
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-12-10 14:53:17 -0800

Refresh of udpong

---

diff --git a/examples/udpong.c b/examples/udpong.c
index 12f0e5b..3b0f961 100644
--- a/examples/udpong.c
+++ b/examples/udpong.c
@@ -375,11 +375,22 @@ static int run_test(void)
 {
 	int ret, i;
 
+printf("send start msg 1\n");
 	msg.op = msg_op_start;
 	ret = client_send_recv(&msg, CTRL_MSG_SIZE, 1000);
 	if (ret != CTRL_MSG_SIZE)
 		goto out;
 
+	printf("send start msg 2\n");
+	ret = client_send_recv(&msg, CTRL_MSG_SIZE, 1000);
+	if (ret != CTRL_MSG_SIZE)
+		goto out;
+	printf("send start msg 3\n");
+	ret = client_send_recv(&msg, CTRL_MSG_SIZE, 1000);
+	if (ret != CTRL_MSG_SIZE)
+		goto out;
+	printf("starting test\n");
+
 	msg.op = echo ? msg_op_echo : msg_op_data;
 	gettimeofday(&start, NULL);
 	for (i = 0; i < transfer_count; i++) {
@@ -429,8 +440,10 @@ static int client_connect(void)
 		rs_close(rs);
 	}
 
+	printf("send login msg\n");
 	msg.op = msg_op_login;
 	ret = client_send_recv(&msg, CTRL_MSG_SIZE, 1000);
+	printf("login %d %s\n", ret, strerror(errno));
 	if (ret == CTRL_MSG_SIZE)
 		ret = 0;
